docs/e2e.md

```diff
@@ -0,0 +1,152 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# End to End
+
+RamenDR end-to-end (e2e) tests validate various scenarios for regional DR
+using predefined workloads and deployment methods.
+
```

Comment:
I think we need this structure, preparing the planned content.  What we have now in this change is only the part of running the tests. The next sections will be added later.

- End to End testing
  - Running End to End tests
    - Preparing a config.yaml file
    - Run All E2E Tests
    - Run Specific E2E Tests
    - List of tests
    - Using multiple config files
  - Extending the e2e tests  
    - Adding new test
    - Adding new workload
    - Adding new deployer
  - Internals